cmd/notary/keys.go

```diff
@@ -322,11 +307,11 @@ func (k *keyCommander) keysRestore(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	ks, err := k.getKeyStores(config, true)
+	ks, err := k.getKeyStores(config, true, true)
```

Comment:
Non-blocking - just wondering if this should actually create a backup - `keysImport` does not, and if you're restoring you already have a backup in terms of a zip file.